src/infrastructure/figma/figma-service.test.ts

```diff
@@ -146,13 +147,18 @@ describe('FigmaService', () => {
 			const node = generateChildNode({ id: nodeId });
 			const designId = generateFigmaDesignIdentifier({ nodeId });
 			const credentials = generateFigmaOAuth2UserCredentials();
+			const figmaUser = generateFigmaUser();
```

Comment:
**nitpick:** Please, see https://github.com/atlassian-labs/figma-for-jira/pull/229/files#r1585903139.